gaps/gap-25_golem_certificates/gap-25_golem_certificates.md

```diff
@@ -0,0 +1,195 @@
+- `signManifest` the key can be used to sign payload manifests (defined in GAP-5) /manifest will not support Golem certificates right now, it requires some update to GAP-5 and the implementation/
+- `signNode` the key can be used to sign a node descriptor (defined in GAP-XX)
+- `all` the key can be used without restrictions, including use cases that are defined after the certificate was created
+
+##### Permissions
+
+Details of how the holder of the certificate is permitted to use the Golem network. The content of this property is governed by the Permissions schema.
+##### Signature
+
+This object contains the details of the signature validating the subject and providing integrity protection for properties in the `certificate` object. It defines the following properties:
+- `algorithm` the cryptographic algorithm that was used to create the signature, it requires two details:
+  - `hash` the hash function used to create the fingerprint of the signed data
+  - `encryption` the cryptographic algorithm used to encrypt the fingerprint
+- `signer` for certificates the signer can the string `self` meaning that the certificate was self signed (aka Root certificate) or an object containing a signed certificate. We opted to include the full certificate as it is easier to handle and understand when opening the JSON certificate in a text editor. We do not anticipate long certificate chains where this would a problem and we think that requiring the certificate to be accessible via the internet would limit the usage for individuals.
+- `value` the hexadecimal encoded string representation of the encrypted fingerprint optionally with a "0x" prefix, it can be decrypted via the signer's public key for verification
+
+#### Permissions
+
+This schema defines the available permission control in the Golem network. It can be found in the `schemas/v1` folder or can be accessed at `https://golem.network/schemas/v1/permissions.schema.json`.
+
+Permissions have two forms:
+- the string `all` which means that the subject is permitted to use all capabilities (present and future) of the Golem network
+- an object containing the detailed permissions of the subject, this object contains the following properties:
+  - `outbound` property contains the detailed permissions related to the `outbound` feature
+    - a string value `unrestricted` means that the subject can use the `outbound` feature without restriction
+    - an object containing information about the feature restrictions, it contains:
+        - `urls` property, which contains an array of URIs that the subject can access via the `outbound` feature. This list has the same semantics as `golem.srv.comp.manifest.net.inet.out.urls` property defined in [GAP-4 Computation Manifest](https://github.com/golemfactory/golem-architecture/blob/master/gaps/gap-4_comp_manifest/gap-4_comp_manifest.md#gcrs-golemsrvcompmanifest-namespace)
+
+
```

Comment:
May I kindly ask for a simple certificate here, for illustration?

Reply:
Uploaded another example for the restricted outbound case where a list of urls is present.
They are in the `examples` folder.